02-image-classification.Rmd

```diff
@@ -11,165 +11,190 @@ cat(author)
 
 ## Lab Overview {-}
 
-Lorem ipsum
+Large-scale remote sensing requires large-scale data storage and computation, which are often limiting factors for studies with large spatial extent or temporal repetition. One solution to the challenge of big data is cloud computing, which allows data storage and processing tasks to happen on remote supercomputing systems (i.e. not on your personal laptop). Google Earth Engine (GEE) provides an interface with which users can efficiently interact with large spatial datasets, including the entire Landsat satellite image archive.
```

Comment:
This should read "Wide area remote sensing requires significant data storage and computation..."